### ISSUE TITLE
Dual screen settings issue

### ISSUE THREAD

**Issue by yanterrien** [muse-sequencer/muse#610]
Dual screen configuration. Ubuntu. The main screen is on a laptop, the secondary screen is above the main screen.
Issue: Muse always opens on the secondary screen at 0,0 no matter what is in the settings.
Issue: when Muse is on the main screen, it goes on the secondary screen each time a project is open.

**Comment by terminator356** [comment]
Hello, sorry for the late reply, I finally got a second monitor running here...
I could not find any issue. I tried "non-unified" screens, side-by-side and secondary on top of primary.
Be sure that your laptop screen really is designated as 'primary', for example in KDE
 System Settings > Display and Monitor.

MusE has always been kind of funny with respect to remembering where it was last placed.
It will remember if a project is opened, moved around and then saved.
It will NOT remember if a blank default project is showing and has been moved around -
 it must be saved first.

Fear not, though. The position - the blank project default - can be FORCED to remember.
First, move MusE to the new desired screen location.
Then open MusE Settings > Global Settings > Views tab and click the 'Main Window' 'Cur(rent)' button.
Notice the coordinate value boxes change to the new location.
Now click 'OK' or 'Apply'.
The next time MusE is opened it should appear at that location.
MusE might of course move again if a project is opened. For an open project
 as mentioned above, simply moving MusE, then saving, should work OK.

Tested: MusE remembered where I left it, with the above caveats in mind.
Hope that helps.
